Clinical-Genomics/scout
# Do not include breakpoints in verification email of SNVs

Fix #2026. It doesn't actually remove the fields in the email, but doesn't assign a value to them.

**How to test**:
1. Install the branch on scout stage
1. Tell prod team in that moment that they're going to get a verification request from you, but it's just for testing.
1. Order a verification for a SV variant
1. Make sure that in the email breakpoint 1 and 2 are "-"

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
